utils.go

```diff
@@ -747,3 +748,5 @@ func HasTaint(node v1.Node, name string) bool {
 	}
 	return false
 }
+
+
```

Comment:
Remove extra lines?

Reply:
I think if this is an issue let's get some tooling to automate it and something in CI to enforce formatting.

Reply:
Not an issue... But I don't think we should leave an intentional new line at the end of the file... It's good that if we find to remove them

Reply:
Not 100% sure but I think we can integrate golangci-lint here as well... That usually complains about extra new-lines in the file

Reply:
Maybe talking past each other but if it isn't an issue then why would it come up as a review comment?

What's the consequence aside from neatness?

For me an "issue" is something that has a consequence. 

I think if it's just a neatness thing let's preface a comment with [neatness] and it it's a super tiny consequence lets say something like [cruft]. Or we can come up with a scheme.

Anyway, I'll make the changes cause I need to make some others.

Reply:
So we should not be holding ourselves to a standard and just let the neatness of code slide away?
I see it as an issue not a major one but yeah worth mentioning out in a review comment. A comment not change request

Reply:
I don't think we need to let messiness slide but I do think there are limits to things that make it to review comments. Lines at the end of a code file don't particularly bother me, though if there's a reason that it should I'm open to that.

My view is that stuff like clipping end of file should be handled automatically. Markdownlint for example handles a lot of that sort of stuff for markdown. If it can't be automated and we care, then we should let a linter catch it.

In the rare case that it's important and we can't find automation for it, I think marking it as non-blocking is helpful.

Perhaps I misunderstood but "Remove extra lines?" feels like a request to change. Tone is difficult in text, which is why I'm suggesting marking non-blocking stuff explicitly.

Reply:
> I do think there are limits to things that make it to review comments.

I do not feel the same

Reply:
Just tested...  even the `go fmt ./...` would remove the extra lines... We don't have the linter for the repo yet and we might put it in the backlog. And these small things are good to be removed as soon as possible so we don't create clutter with lots of them

Reply:
I've added a linter.

Reply:
Running go fmt should be standard practice - and leaving a formatting issue in the codebase causes problem later on, when you go fmt in the future and it changes files you haven't touched

Reply:
Genuinely think that the time taken to have this conversation is enough evidence that a linter is important.